jboss-kie-kieserver/tests/bats/jboss-kie-kieserver-jms.bats

```diff
@@ -237,4 +239,51 @@ teardown() {
     echo "status = ${status}"
     echo "output = ${output}"
     [ "${status}" = "2" ]
-}
\ No newline at end of file
+}
+
+@test "Verify the KieExecutorMDB default value " {
```

Comment:
isn't the other tests doing it already?

Reply:
No, it mean configuration without values, changing the title of the test